packages/e/element/package.yml

```diff
@@ -44,9 +44,22 @@ setup      : |
     jq '.update_base_url = null | .posthog = null | .show_labs_settings = true' \
         element.io/release/config.json > config.json
 
+    tmp=$(mktemp)
+    # Create an unpacked distribution, allowing us to more easily see changes to shipped files
```

Comment:
While I like this from a review-ability perspective, the last time I checked it significantly impacted the startup time of the application for users with slower drives (ie. not NVMe).

We should probably hash out some conventions on how we want to bundle NodeJS applications, as that currently seems to be all over the place.

Reply:
I tested a different app and could not create a scenario where the asar was ever faster than having it extracted like this. Granted, I used a NVMe drive which is obviously great at random file IO, but in every scenario it (slightly) beat the asar for startup time. Considering that the majority of users should be on SSDs by now I'm not really sure if we want to hold back things like this for a minority of users?

Reply:
An NVMe drive is still orders of magnitude faster with random IO than a SATA SSD, and I don't think NVMe drives are in the majority.

I don't have a strong opinion either way, but as I said in my previous comment: we should probably hash out some conventions on how we want to bundle NodeJS applications so packaging is somewhat consistent and follow that.

Reply:
Not doing that anymore.